src/DependencyInjection/AnswearWideEyesExtension.php

```diff
@@ -26,9 +26,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $definition = $container->getDefinition(ConfigProvider::class);
         $definition->setArguments(
             [
-                $config['apiUrl'],
+                $config['similarApiUrl'],
+                $config['searchByImageApiUrl'],
                 $config['publicKey'],
-                $config['requestTimeout'],
+                $config['similarRequestTimeout'],
+                $config['searchByImageRequestTimeout'],
```

Comment:
Dodaję dodatkowe urle i timeouty dla searchByImage - okazuje się, że o te rzeczy mamy pytać pod innym adresem. 
Rózne timeputy wynikają z tego, że na similar dajemy jedną sekunę (ustalone wczesniej, kwestia tego, żeby nie spowalniało nam endpotinu produktu), a dla searchByImage to za mało - nigdy w tym czasie nie dostajemy odpowiedzi.

Reply:
will be better to have  array config. ex:
```json
{
   publicKey:
   similar:
      apiUrl:
      timeout:
   searchByImage:
      apiUrl:
      timeout:
}
```

#whatever

Reply:
Great idea, I'm gonna include it in my next PR